optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/AbstractXlsxSolutionFileIO.java

```diff
@@ -59,24 +59,25 @@
     protected static final Pattern VALID_CODE_PATTERN = Pattern.compile("(?U)^[\\w\\-\\.\\/\\(\\)]+$");
 
     protected static final DateTimeFormatter DAY_FORMATTER
-            = DateTimeFormatter.ofPattern("E yyyy-MM-dd", Locale.ENGLISH);
+        = DateTimeFormatter.ofPattern("E yyyy-MM-dd", Locale.ENGLISH);
```

Comment:
@MusaTalluzi Looks like you have continuation indent set to 4 in your IDE. It should be 8.